lib/archethic.ex

```diff
@@ -55,8 +48,6 @@ defmodule Archethic do
       do_send_transaction(tx)
     else
       P2P.authorized_and_available_nodes()
-      |> Enum.filter(&Node.locally_available?/1)
-      |> P2P.nearest_nodes()
```

Comment:
This selection should stay here I think as when we want to forward transaction, we should do it on local and near node to be fast